edx_solutions_api_integration/users/serializers.py

```diff
@@ -206,3 +207,20 @@ def get_course(self, enrollment):
             ), None
         )
         return course_overview
+
+
+class ExtendedOrganizationSerializer(BasicOrganizationSerializer):
+    """ Serializer for Organization with mobile apps and themes data """
+    mobile_apps = MobileAppSerializer(many=True)
+    themes = serializers.SerializerMethodField()
```

Comment:
one organization is going to have one active theme so name of this fields should be `theme` and values of it should be a dict representing active theme of the organization instead of a list.